src/plugins/orangehrmPimPlugin/Api/EmployeeContactDetailsAPI.php

```diff
@@ -238,6 +239,8 @@ public function getValidationRuleForUpdate(): ParamRuleCollection
                     self::PARAMETER_OTHER_EMAIL,
                     new Rule(Rules::STRING_TYPE),
                     new Rule(Rules::EMAIL),
+                    new Rule(Rules::ENTITY_UNIQUE_PROPERTY, [Employee::class, 'otherEmail', $this->getEntityPropertiesForEmailRule()]),
+                    new Rule(Rules::ENTITY_UNIQUE_PROPERTY, [Employee::class, 'workEmail']),
```

Comment:
Note: Employee cannot switch the emails around easily. For example if they have `test@work.com` and `test@other.com` for Work and Other respectively, they can't switch the values and submit. This behaviour is present in 4.x also

![image](https://user-images.githubusercontent.com/97493184/163970672-d2a5043c-ef35-4cbd-b085-ebc42f35c729.png)

![image](https://user-images.githubusercontent.com/97493184/163971378-cdce7dc5-7756-42c1-86a9-e8ed757151d8.png)